functions/go/apply-setters/metadata.yaml

```diff
@@ -0,0 +1,14 @@
+image: gcr.io/kpt-fn/apply-setter
+description: Apply setter values on resource fields. Setters serve as parameters for template-free setting of field values.
```

Comment:
Are you saying this is automatically populated from usage doc? Where's the logic for that?